Api/Data/AmwalButtonConfigInterface.php

```diff
@@ -335,4 +334,16 @@ public function getCartId(): ?string;
      */
     public function setCartId(?string $cartId): AmwalButtonConfigInterface;
 
+
+    /**
+     * @return array|null
+     */
+    public function getErrorMessage(): ?array;
+
+    /**
+     * @param array|null $errorMessage
```

Comment:
We can type this slightly more specific by defining what we expect in the array:
```suggestion
     * @param (string|Phrase)[]|null $errorMessage
```

Based on: https://github.com/php-fig/fig-standards/blob/211063eed7f4d9b4514b728d7b1810d9b3379dd1/proposed/phpdoc.md#arrays